docker-compose.yml

```diff
@@ -4,8 +4,9 @@ services:
   # No db server since uses MySQLite database.
 
   see_stuff:
-    build: .
-    image: "rails_see_stuff_container"
+    # build: .
+    # image: "rails_see_stuff_container"
```

Comment:
I suggest deleting the commented-out lines.

Reply:
Deleted both comment lines.

Reply:
Confirmed the commented-out lines are gone.